cmake/UnitTests.cmake

```diff
@@ -432,6 +431,7 @@ list(APPEND CPU_OPENCL_FAILED_TESTS "syncthreadsExitedThreads") # Timeout
 list(APPEND CPU_OPENCL_FAILED_TESTS "hipMultiThreadAddCallback") # SEGFAULT
 
 # iGPU OpenCL Unit Test Failures
+list(APPEND IGPU_OPENCL_FAILED_TESTS "Unit_hipMultiThreadStreams1_AsyncSame") # invalid free()
```

Comment:
FYI, this seems to be a flaky case. On my Intel iGPU the test passes.

Reply:
It's a race condition addressed in #657